webextension/tools.js

```diff
@@ -79,6 +84,14 @@ class Tools {
         return chrome.storage.sync && !Tools.isFirefox() ? chrome.storage.sync : chrome.storage.local;
     }
 
+    static doNotSupportOnUrl(url) {
+        return url.match(unsupportedSitesRegex);
```

Comment:
please use 2 space indent here, too

Reply:
Okay, no problem. For consistency, we should use .editorconfig file setting http://editorconfig.org/